bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -1301,10 +1301,31 @@ class AudioManager {
     const { outputDeviceId } = this.bridge;
 
     if (outputDeviceId && typeof audioAlert.setSinkId === 'function') {
-      return audioAlert.setSinkId(outputDeviceId).then(() => audioAlert.play());
+      return audioAlert
+        .setSinkId(outputDeviceId)
+        .then(() => this.canAutoplayAudioElement(audioAlert));
     }
 
-    return audioAlert.play();
+    return this.canAutoplayAudioElement(audioAlert);
+  }
+
+  canAutoplayAudioElement(element) {
+    return new Promise((resolve) => {
+      if (!(element instanceof HTMLMediaElement)) {
+        // Provided element is not a valid audio/video element.
+        resolve(false);
+        return;
+      }
+
+      element.play().then(() => {
+        resolve(true);
+      }).catch(() => {
+        this.handlePlayElementFailed({
```

Comment:
This should not be called here. The failure handler has to be used at the discretion of the caller. The only time it's supposed to be called in audio-manager is during the audio join procedure (i.e.: when playing the microphone output element fails) as it'll trigger the autoplay handling modal.

Doing it here will possibly cause unintended UI flows.